sources/exception.asciidoc

```diff
@@ -157,29 +157,15 @@ Reasons for raising this exception include:
 * The extracted type is not marked with `@ExtractedValue`
 * The `@ExtractedValue` annotation is given more than once for one value extractor type
 
-[role="api"]
```

Comment:
I've removed the includes to be consistent with what we're doing for the other exceptions.

@gsmet Should we sprinkle some tck-testable markers over the examples given for exception cases?